src/darker/command_line.py

```diff
@@ -105,6 +107,14 @@ def parse_command_line(argv: List[str]) -> Tuple[Namespace, DarkerConfig, Darker
     #    This is used to find out differences between the effective configuration and
     #    default configuration values, and print them out in verbose mode.
     parser_with_original_defaults = make_argument_parser(require_src=True)
+    invalid_src_paths = [src for src in args.src if src.startswith("-")]
```

Comment:
Can you just check something like `os.path.exists(...)`? This seems like it would cover more use cases than just handling them one-by-one as things get messed up as is referenced in #147

Reply:
Indeed! This would also be more in line with what Black does. For non-existing files on the command line it terminates with
```
$ black nonexisting.py
Usage: black [OPTIONS] SRC ...
Try 'black -h' for help.

Error: Invalid value for 'SRC ...': Path 'nonexisting.py' does not exist.
```
and for missing directories
```
$ black nonexisting/
Usage: black [OPTIONS] SRC ...
Try 'black -h' for help.

Error: Invalid value for 'SRC ...': Path 'nonexisting/' does not exist.
```
I'll modify the patch to do the same in Darker as well.

Reply:
...although in Darker's case it's a bit more nuanced than that. Let's say you do
```
$ darker --revision master..HEAD oldfile.py
```
but you've deleted `oldfile.py` in your working tree. Darker should still check the file contents between `master` and the last commit (`HEAD`).

This is still doable, just requires a bit more thinking on where to check for existence of files and directories.

Reply:
In [darker.git.git_get_modified_files()](https://github.com/akaihola/darker/blob/508a6ec169d9add0e6c1123f0f20f7fc6829a3f1/src/darker/git.py#L169-L175), Darker calls
```
git diff --name-only --relative <rev> -- <path(s)>
```
for changed files, and additionally
```
git ls-files --others --exclude-standard -- <path(s)>
```
if comparing against the working tree so newly created Python files not yet tracked with Git are also processed.

Unfortunately, `git diff` gives empty output both for unchanged and non-existing file paths. So we'd need to introduce a third Git invocation to check which files are missing from `rev2` (if `rev2` is not `WORKTREE`).

In case `rev2` is `WORKTREE`, a simple filesystem existence check does the trick.

Reply:
@levouh I implemented the existence check both for the working tree (filesystem) and arbitrary Git commits. How does it look to you now?

Reply:
@levouh, what about this scenario:

- `file.py` exists in `commit1`
- `file.py` is removed in `commit2`
- user does `darker --diff --revision commit1..commit2`

Should this exit with an error or succeed silently? Surely there's nothing to report since there are no "modified lines" left in a non-existing file.

With the new `--stdout` option (#165), this surely shouldn't succeed with empty output, since that would indicate that an empty `file.py` *does* exist in `commit2`.

Reply:
@levouh, FYI, I decided to go with the design in which existence in `rev2` is validated when `--revision=rev1..rev2` is specified.